packages/main/src/themes/Dialog.css

```diff
@@ -46,19 +46,64 @@
 	position: relative;
 }
 
+:host([state="Error"]) ::slotted([slot="header"][ui5-bar]),
 :host([state="Error"]) .ui5-popup-header-root {
-	box-shadow: var(--_ui5_dialog_header_error_state_box_shadow);
+	box-shadow: none;
 }
 
+:host([state="Error"]) .ui5-popup-header-root::before {
+	content: "";
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	height: 0.0625rem;
```

Comment:
try to reduce the repetitive code and use the appropriate logical CSS property to position the element